File: src/main/java/net/ndrei/teslacorelib/test/CreativeGeneratorEntity.java
package net.ndrei.teslacorelib.test;

import net.ndrei.teslacorelib.tileentities.ElectricGenerator;

/**
 * Created by CF on 2016-12-27.
 */
public class CreativeGeneratorEntity extends ElectricGenerator {
    public CreativeGeneratorEntity() {
        super(-2);
    }

    @Override
    protected long consumeFuel() {
        return 50000;
    }
}

File: src/main/java/net/ndrei/teslacorelib/test/CreativeGeneratorEntity.kt
package net.ndrei.teslacorelib.test

import net.ndrei.teslacorelib.tileentities.ElectricGenerator

/**
 * Created by CF on 2017-06-27.
 */
class CreativeGeneratorEntity : ElectricGenerator(-2) {
    override fun consumeFuel(): Long {
        return 50000
    }
}